persistence/core/src/main/scala/com/lightbend/lagom/internal/persistence/testkit/package.scala

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (C) 2016-2018 Lightbend Inc. <https://www.lightbend.com>
+ */
+package com.lightbend.lagom.internal.persistence
```

Comment:
Is this supposed to be in `src/main' ?

I'm guessing not, but you had to put it here because of some existing, unrelated code depending on this which required extra refactoring.

Let's add a `TODO` in a scaladoc stating we should move this.

Reply:
Right now this is only used internally in our own tests, rather than exposed for end users.

I didn't think it made sense to put this into `src/main` because then it would always be in the classpath at runtime. There are a few classes like that already in the `javadsl`/`scaladsl` persistence modules, but I didn't want to make the problem worse. The organization of test kit classes really needs to be rethought.

I don't think I'd want to just move it as is, and I'm not really sure what form it should ultimately take.

Reply:
Oh, I overlooked that it's in `.internal` package. 👍 

> The organization of test kit classes really needs to be rethought

Yep. OK, let's leave this as is and we'll take over when fixing https://github.com/lagom/lagom/issues/795.